File: Ex31/app/src/main/java/kr/hs/ex31/MainActivity.java
package kr.hs.ex31;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.RadioButton;

public class MainActivity extends AppCompatActivity {
    RadioButton two, three;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button btnNewActivity = (Button) findViewById(R.id.new_btn);
        two = (RadioButton) findViewById(R.id.two);
        three = (RadioButton) findViewById(R.id.third);

        btnNewActivity.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (two.isChecked()){
                    Intent intent = new Intent(getApplicationContext(), SecondActivity.class);
                    startActivity(intent);
                }
                if(three.isChecked()){
                    Intent intent = new Intent(getApplicationContext(), ThirdActivity.class);
                    startActivity(intent);
                }

            }
        });
    }
}

File: Splash/app/src/main/java/kr/hs/splash/MainActivity.kt
package kr.hs.splash

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView

class MainActivity : AppCompatActivity() {
    private val ImgIds : Array<Int> = arrayOf(R.id.one, R.id.two, R.id.three, R.id.four, R.id.five, R.id.six, R.id.seven, R.id.eight, R.id.nine)
    private var ImgArray = Array<ImageView?>(9) { null }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        for(i in ImgIds.indices){
            ImgArray[i] = findViewById(ImgIds[i])
        }
        for(i in ImgIds.indices){
            val index = i;

            ImgArray[index]!!.setOnClickListener {
                val intent = Intent(this, ImageinsideActivity::class.java)
                intent.putExtra("data", index.toString())
                startActivity(intent)
            }
        }



    }
}